packages/vscode-extension/src/dependency/DependencyChecker.ts

```diff
@@ -123,21 +123,32 @@ export class DependencyChecker implements Disposable {
   }
 
   public async checkCocoaPodsInstalled() {
-    const installed = await checkIfCLIInstalled("pod --version", {
-      env: { ...process.env, LANG: "en_US.UTF-8" },
+    const env = { ...process.env, LANG: "en_US.UTF-8" };
+    let isCocoaPodsInstalled = await checkIfCLIInstalled("pod --version", {
+      env,
     });
+
+    // If `pod --version` fails, try `bundle exec pod --version` as a second attempt
+    if (!isCocoaPodsInstalled && workspace.workspaceFolders?.length) {
```

Comment:
Nit: can we please make it more explicit? `workspace.workspaceFolders?.length > 0`